### PR TITLE
sci-physics/pyhepmc: new package, add 2.11.0

### PR DESCRIPTION
**Pull request by APN-Pucky**

### CODE REVIEW

File: sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,17 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Pythonic wrapper for the HepMC3 C++ library."
```
Comment (AndrewAmmerlaan) [inline sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild:7]: Should we RDEPEND on hepmc?

Comment (APN-Pucky) [reply sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild:7]: No it compiles it by itself at a fixed version through some python build system. But i should add a BDEPEND on cmake.

Comment (AndrewAmmerlaan) [reply sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild:7]: It bundles hepmc? Is it feasible to make it use the system version instead?

Comment (AndrewAmmerlaan) [reply sci-physics/pyhepmc/pyhepmc-2.11.0.ebuild:7]: That makes sense

**Pull request closed without merging**